invoice2data/input/tesseract.py

```diff
@@ -19,12 +22,13 @@ def to_text(path):
     # Check for dependencies. Needs Tesseract and Imagemagick installed.
     if not spawn.find_executable('tesseract'):
         raise EnvironmentError('tesseract not installed.')
-    if not spawn.find_executable('convert'):
+    if not spawn.find_executable('convert'):  # Please remember that on Windows exists C:\Windows\System32\convert.exe and have the same name as ImageMagick tool
         raise EnvironmentError('imagemagick not installed.')
 
+    is_shell_active = platform.system() == "Windows"  # ImageMagick on Windows require shell=True for working
     # convert = "convert -density 350 %s -depth 8 tiff:-" % (path)
     convert = ['convert', '-density', '350', path, '-depth', '8', 'png:-']
-    p1 = subprocess.Popen(convert, stdout=subprocess.PIPE)
+    p1 = subprocess.Popen(convert, stdout=subprocess.PIPE, shell=is_shell_active)
```

Comment:
Adding this to all platforms shouldn't cause any issues and would save the extra condition.

Reply:
ok, I will change to:

`p1 = subprocess.Popen(convert, stdout=subprocess.PIPE, shell=True)`